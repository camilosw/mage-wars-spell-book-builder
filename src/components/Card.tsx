import * as React from 'react';
// import CardType from '../types/Card';

const Card: React.SFC<{}> = () => <div className="Card">Card</div>;

export default Card;
