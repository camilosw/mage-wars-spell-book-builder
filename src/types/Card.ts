interface Card {
  code: string;
  name: string;
  type: string;
  subtype?: string;
  typeName: string;
  school: string;
  level: string;
  cast: string;
  reveal?: string;
  max: number;
  count: {
    [name: string]: number;
  };
}

export default Card;
