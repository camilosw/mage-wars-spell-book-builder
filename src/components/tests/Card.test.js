import React from 'react';
// import cards from '../../data/cards';
import Card from '../Card';
// import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

it('render Card with data', () => {
  const component = renderer.create(<Card />);
  expect(component.toJSON()).toMatchSnapshot();
});

// describe('<Card />', () => {
//   it('render Card with data', () => {
//     const card = cards[2];
//     // shallow(<Card data={card} />);
//   });
// });
