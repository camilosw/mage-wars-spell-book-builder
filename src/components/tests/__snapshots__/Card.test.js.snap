// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Card with data 1`] = `
<div
  className="Card"
>
  Card
</div>
`;
